packages/cli/src/commands/pw-test.ts

```diff
@@ -0,0 +1,365 @@
+    const repoInfo = getGitInformation(project.repoUrl)
+    const ciInfo = getCiInformation()
+    // TODO: ADD PROPER RETRY STRATEGY HANDLING
+    // const testRetryStrategy = this.prepareTestRetryStrategy(retries, checklyConfig?.cli?.retries)
+
+    const runner = new TestRunner(
+      config.getAccountId(),
+      projectBundle,
+      checkBundles,
+      Session.sharedFiles,
+      location,
+      timeout,
+      verboseFlag,
+      record,
+      repoInfo,
+      ciInfo.environment,
+      // NO NEED TO UPLOAD SNAPSHOTS FOR PLAYWRIGHT TESTS
+      false,
+      configDirectory,
+      // TODO: ADD PROPER RETRY STRATEGY HANDLING
+      null, // testRetryStrategy
+    )
+
+    runner.on(Events.RUN_STARTED,
+      (checks: Array<{ check: any, sequenceId: SequenceId }>, testSessionId: string) =>
+        reporters.forEach(r => r.onBegin(checks, testSessionId)),
+    )
+
+    runner.on(Events.CHECK_INPROGRESS, (check: any, sequenceId: SequenceId) => {
+      reporters.forEach(r => r.onCheckInProgress(check, sequenceId))
+    })
+
+    runner.on(Events.MAX_SCHEDULING_DELAY_EXCEEDED, () => {
+      reporters.forEach(r => r.onSchedulingDelayExceeded())
+    })
+
+    runner.on(Events.CHECK_ATTEMPT_RESULT, (sequenceId: SequenceId, check, result, links?: TestResultsShortLinks) => {
+      reporters.forEach(r => r.onCheckAttemptResult(sequenceId, {
+        logicalId: check.logicalId,
+        sourceFile: check.getSourceFile(),
+        ...result,
+      }, links))
+    })
+
+    runner.on(Events.CHECK_SUCCESSFUL,
+      (sequenceId: SequenceId, check, result, testResultId, links?: TestResultsShortLinks) => {
+        if (result.hasFailures) {
+          process.exitCode = 1
+        }
+
+        reporters.forEach(r => r.onCheckEnd(sequenceId, {
+          logicalId: check.logicalId,
+          sourceFile: check.getSourceFile(),
+          ...result,
+        }, testResultId, links))
+      })
+
+    runner.on(Events.CHECK_FAILED, (sequenceId: SequenceId, check, message: string) => {
+      reporters.forEach(r => r.onCheckEnd(sequenceId, {
+        ...check,
+        logicalId: check.logicalId,
+        sourceFile: check.getSourceFile(),
+        hasFailures: true,
+        runError: message,
+      }))
+      process.exitCode = 1
+    })
+    runner.on(Events.RUN_FINISHED, () => reporters.forEach(r => r.onEnd()))
+    runner.on(Events.ERROR, (err) => {
+      reporters.forEach(r => r.onError(err))
+      process.exitCode = 1
+    })
+    await runner.run()
+    }
+
+    static async createPlaywrightCheck(args: string[], runLocation: keyof Region, dir: string): Promise<PlaywrightSlimmedProp> {
+      const parseArgs = args.map(arg => {
+        if (arg.includes(' ')) {
+          arg = `"${arg}"`
+        }
+        return arg
+      })
+      const input = parseArgs.join(' ') || ''
+      const inputLogicalId = input.replace(/[^a-zA-Z0-9]/g, '-').toLowerCase().substring(0, 50)
```

Comment:
You can use `cased(input, 'kebab-case')` from `packages/cli/src/sourcegen/case.ts` instead.

Reply:
Or `kebabCase(input)` from the same place if you prefer. It's essentially the same thing.